gap/Magic.gd

```diff
@@ -48,6 +48,10 @@
 #!     supplementary files (such as CSS style files) into your doc directory
 #!     (see <Ref Func='CopyHTMLStyleFiles' BookName='gapdoc'/>).
 #! </Item>
+#! <Item>
+#!     It can create a maketest.g file which tests that all of the examples
+#!     in the generated documentation perform as documented.
+#! </Item>
```

Comment:
I am really torn on whether to officially document this feature (I realize it is already mentioned in the Tutorial... *sigh*... but see issues #114, #117 and #155)

Reply:
Since this feature has now been deprecated, we should rather not document it.